src/main/resources/db_scripts/liquibase/v0.4.4.refactor-job-parameters.sql

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2018 ABSA Group Limited
```

Comment:
You could already add the next script for v0.4.5 to drop the variables, maps, key_value_pairs columns, but not add it in the changelog, but that's up to you. Either way someone has to do it in the next version

Reply:
I will do it in its own PR